syntax = "proto2";

package drpc;

option cc_generic_services=true;

message EchoRequest {
    required string message=1;
}

message EchoResponse {
    required string message=1;
}

service EchoServer {
    rpc Echo(EchoRequest) returns(EchoResponse);
}